assets/js/atomic/blocks/product-elements/image/test/block.test.js

```diff
@@ -62,7 +62,10 @@ describe( 'Product Image Block', () => {
 	describe( 'with product link', () => {
 		test( 'should render an anchor with the product image', () => {
 			const component = render(
-				<ProductDataContextProvider product={ productWithImages }>
+				<ProductDataContextProvider
+					product={ productWithImages }
+					isLoading={ false }
```

Comment:
Can you clarify why `isLoading` is hardcoded to `false` in the context of this test, and why was it needed to be added as a fix for this issue? 🤔

Reply:
> Can you clarify why isLoading is hardcoded to false in the context of this test

Sure! I added `isLoading` prop to each test, as I realised it's a required prop for `ProductDataContextProvider` and TypeScript checker complained about that.
And considering the tests are checking the behaviour of component with images loaded already, I checked the code and `isLoading` passed as `false` seems to set such condition. Also, there's no default value for `isLoading`, so before adding that to tests, it was implicitly falsy.
But I may be missing something, so I'm all ears if you see that may be troublesome! 👂 

> and why was it needed to be added as a fix for this issue? 🤔

TS fixes/improvements were added as an extra to this PR and are not required as a fix for issue. I gathered them [in a separate commit](https://github.com/woocommerce/woocommerce-blocks/pull/7651/commits/4763d7f182198077b06bfaa2844a6a4227ccdd06), but maybe I should've added it as a separate PR to avoid confusion. Let me know the way forward 🙌